sources/core/objects/mobs/Human.cpp

```diff
@@ -427,6 +429,14 @@ void Human::Live()
                 }
             }
         }
+        if(plasma > 35)
+        {
+            if (CalculateHealth() >= 0)
+            {
+                holder->RemoveGase(atmos::PLASMA, 1);
+                ApplyToxinsDamage(50);
```

Comment:
Same here - instead of `50` it is better to make a special constant with a descriptive name.